lua/tardis/sh_interiors.lua

```diff
@@ -154,9 +251,22 @@ local name_overrides = TARDIS_OVERRIDES.Names or {}
 
 local default_category = TARDIS_OVERRIDES.MainCategory or "Doctor Who - TARDIS"
 
-TARDIS.MetadataTemplates = TARDIS.MetadataTemplates or {}
+function TARDIS:FullReloadInteriors()
```

Comment:
Could you describe why this is needed? The code should be designed in such a way that auto reload can reload everything automatically without having to do a full reset like this every time. It's fine if it's required, but I want to make sure we're not hiding a problem elsewhere by doing this

Reply:
- Interiors are not only dependent on each other, but also on templates
- The templates have to be merged strictly after all `t.Base` tables are merged, other behaviour breaks their purpose
- When you changed the interior, AddInterior() would not call merging the templates
- When you changed the template, it would completely wreck up the interior

I tried to make an automatic re-merge of this interior in AddTemplate() and AddInterior()
I failed. 

One of the "game-freaks-out" scenarios: 
- Interiors `A` and `B`
- `A` has templates `a1`, `a2` and `a3`; `a3` is an overriding template, `a1` and `a2` are not
- `B.Base=A`
- B inherits `a1`, `a2`, `a3` templates from metadata
- We specify that `a2 = false` in `B` (it should not happen), and add `b2` template

Currently, interiors have chains of 3-4 `T.Base` merges
This whole thing is too complicated to track all the merges and re-make them in the right order

Reply:
Okay, considering this only happens when you've developing interiors anyway and not under normal circumstances it's not a huge deal, just wanted to make sure this wasn't just added as a get out of jail free card, when it could be fixed. Well maybe it still can but this seems like an okay alternative.